### ISSUE TITLE
Build Error

### ISSUE THREAD

**Issue by 2br-2b** [t7ko/jdotxt#22]
Trying to build the program, I get this error:

```
foo@computer:~/uncompiled/fork/jdotxt$ ant
Buildfile: /home/foo/uncompiled/fork/jdotxt/build.xml

init:
    [mkdir] Created dir: /home/foo/uncompiled/fork/jdotxt/build/main
    [mkdir] Created dir: /home/foo/uncompiled/fork/jdotxt/build/test

compile:
    [javac] Compiling 68 source files to /home/foo/uncompiled/fork/jdotxt/build/main
    [javac] /home/foo/uncompiled/fork/jdotxt/src/com/chschmid/jdotxt/gui/JdotxtGUI.java:44: error: package javax.xml.bind does not exist
    [javac] import javax.xml.bind.DatatypeConverter;
    [javac]                      ^
    [javac] /home/foo/uncompiled/fork/jdotxt/src/com/todotxt/todotxttouch/task/JdotxtTaskBagImpl.java:26: error: package javax.xml.bind does not exist
    [javac] import javax.xml.bind.DatatypeConverter;
    [javac]                      ^
    [javac] /home/foo/uncompiled/fork/jdotxt/src/com/todotxt/todotxttouch/task/TaskBagImpl.java:26: error: package javax.xml.bind does not exist
    [javac] import javax.xml.bind.DatatypeConverter;
    [javac]                      ^
    [javac] Note: /home/foo/uncompiled/fork/jdotxt/src/com/chschmid/jdotxt/gui/JdotxtGUI.java uses or overrides a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 3 errors

BUILD FAILED
/home/foo/uncompiled/fork/jdotxt/build.xml:36: Compile failed; see the compiler error output for details.

Total time: 19 seconds
```

I tried building the original program, and I can build it fine.

**Comment by t7ko** [comment]
What are the versions you're building on?

I'm using:
ant = apache-ant-1.9.14
java = jdk1.8.0_171

**Comment by t7ko** [comment]
... and I'm doing it on Windows.

**Comment by 2br-2b** [comment]
I'm on Linux, and I'm running:
```
openjdk version "11.0.8" 2020-07-14
OpenJDK Runtime Environment (build 11.0.8+10-post-Ubuntu-0ubuntu120.04)
OpenJDK 64-Bit Server VM (build 11.0.8+10-post-Ubuntu-0ubuntu120.04, mixed mode, sharing)
```

I'm also running `Apache Ant(TM) version 1.10.8 compiled on May 10 2020`.

**Comment by t7ko** [comment]
Apparently, Java v11 removes those classes under java.xml.bind:

https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist

On the other hand, this package was only there for debug purposes, so I'm removing it now.

**Comment by t7ko** [comment]
should be fixed now with latest merge #25.

**Comment by 2br-2b** [comment]
It works great now! Thanks!
